packages/terra-visually-hidden-text/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Update react/react-dom peer-dependency to latest stable minor release
```

Comment:
This changelog isn't syntax highlighting correctly, but it does preview correctly. Might not be an issue, but I thought it was worth mentioning.

![image](https://user-images.githubusercontent.com/6720991/54888291-0d691080-4e6a-11e9-8da9-eed102d19429.png)

![image](https://user-images.githubusercontent.com/6720991/54888302-24a7fe00-4e6a-11e9-9a24-a7ce84c43f0d.png)

Reply:
This looks like it's syntax highlighting correctly now, it must have just been something in the browser or with github.